packages/@aws-cdk/aws-iam/package.json

```diff
@@ -149,6 +149,9 @@
       "docs-public-apis:@aws-cdk/aws-iam.PrincipalBase.toJSON",
       "docs-public-apis:@aws-cdk/aws-iam.PrincipalPolicyFragment.conditions",
       "docs-public-apis:@aws-cdk/aws-iam.PrincipalPolicyFragment.principalJson",
+      "docs-public-apis:@aws-cdk/aws-iam.PrincipalWithConditions.conditions",
```

Comment:
Please document the offending members instead of silencing the errors.

Reply:
Will do.

I took the silencing of errors for the same/similar fields on `PrincipalBase` and `PrincipalPolicyFragment` to indicate there had been a decision that there was nothing worth documenting about these fields. If that's not the case, I'll add docs.

Reply:
In fact the decision was "we don't want to regress docs going forward but we're not going to fix all instances before introducing the check".

Reply:
So all existing members *should* be documented but we haven't gotten around to it yet.